src/test/libcephfs/test.cc

```diff
@@ -1488,26 +1488,26 @@ TEST(LibCephFS, Btime) {
 
   ASSERT_EQ(ceph_fstatx(cmount, fd, &stx, CEPH_STATX_CTIME|CEPH_STATX_BTIME, 0), 0);
   ASSERT_TRUE(stx.stx_mask & (CEPH_STATX_CTIME|CEPH_STATX_BTIME));
-  ASSERT_EQ(stx.stx_btime, stx.stx_ctime);
-  ASSERT_EQ(stx.stx_btime_ns, stx.stx_ctime_ns);
+  ASSERT_TRUE(stx.stx_ctime.tv_sec == stx.stx_btime.tv_sec &&
+	      stx.stx_ctime.tv_nsec == stx.stx_btime.tv_nsec);
   ceph_close(cmount, fd);
```

Comment:
Do timespecs not have have equality operator you can use instead of calling out the pieces?